must-gather/collection-scripts/logs/gather_logs_backup

```diff
@@ -15,11 +15,11 @@ skip_tls=$8
 mkdir -p "{object_collection_path}"
 echo "[cluster=${cluster}][ns=${ns}] Gathering 'velero backup describe ${backup}'"
 if [ "$timeout" = "0s" ]; then
-    oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero describe backup ${backup} --insecure-skip-tls-verify=${skip_tls} --details" &> "${object_collection_path}/backup-describe-${backup}.txt" &
+    oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "timeout 30s /velero describe backup ${backup} --insecure-skip-tls-verify=${skip_tls} --details" &> "${object_collection_path}/backup-describe-${backup}.txt" &
 else
-    oc -n ${ns} exec --request-timeout=${timeout} $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero describe backup ${backup} --insecure-skip-tls-verify=${skip_tls} --details" &> "${object_collection_path}/backup-describe-${backup}.txt" &
+    oc -n ${ns} exec --request-timeout=${timeout} $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "timeout ${timeout} /velero describe backup ${backup} --insecure-skip-tls-verify=${skip_tls} --details" &> "${object_collection_path}/backup-describe-${backup}.txt" &
```

Comment:
do we need `--request-timeout=${timeout}` here if timeout is getting passed in the velero container?

Reply:
in theory no.

Reply:
From further testing and doc digging.. --request-timeout has very different effects to `timeout` in velero container.

--request-timeout is [The length of time to wait before giving up on a single server request.](https://kubernetes.io/docs/reference/kubectl/kubectl/)

whereas, if the server has responded, request-timeout do not work to kill a stuck velero CLI process.

So I propose we keep both for the `$timeout` defined case.